controllers/postController.js

```diff
@@ -0,0 +1,49 @@
+const bodyParser    = require('body-parser');
+const mongoose      = require('mongoose');
+const Post          = require('../models/post');
+const PostService   = require('../services/PostService');
+const express       = require('express');
+
+
+const router        = express.Router();
+
+const urlencodedParser = bodyParser.urlencoded({ extended: false });
+
+// Connect to database
+mongoose.Promise = require('bluebird');
+mongoose.connect('mongodb://test:test@ds257627.mlab.com:57627/first-blog-mern', { useMongoClient: true });
+
+const postService = new PostService();
+
+
+router.get(['/', '/posts'], (req, res) => {
+    postService.getAllPosts().then(posts => {
+        res.render('posts', {posts: posts});
+    }).catch(() => {
+        res.status(500).send('Service Unavailable')
```

Comment:
500 is Internal Server Error, not Service Unavailable. Also the message `'Service Unavailable'` doesn't say much.
If you catch the err (as the example I provided), you can provide more information